src/examples/java/com/nylas/examples/other/CalendarsExample.java

```diff
@@ -58,22 +66,25 @@ public static void main(String[] args) throws Exception {
 		created.setMetadata(metadata);
 		Calendar updated = calendars.update(created);
 		log.info("Updated: " + updated + " status: " + updated.getJobStatusId());
-		
+
 		String deleteJobStatusId = calendars.delete(updated.getId());
 		log.info("Deleted, deleted job status id: " + deleteJobStatusId);
-		
+
 		JobStatus deleteStatus = account.jobStatuses().get(deleteJobStatusId);
 		log.info("Deletion status: " + deleteStatus);
 
-		availability(calendars);
+		availability(calendars, accountId, calendarId, calendarEmail);
```

Comment:
the accountId and calendarId could be null here. Thinking of the case with a virtual calendar when the name won't be an email. 
the availability() method may want to check those inputs

Reply:
Done, cleaned up the example file as a whole ✅